internal/models/user.go

```diff
@@ -808,6 +786,10 @@ func (u *User) SoftDeleteUser(tx *storage.Connection) error {
 		return err
 	}
 
+	if err := Logout(tx, u.ID); err != nil {
```

Comment:
Checking -  this change is unrelated to the one time token changes and is a general security improvement right?

Reply:
Yeah I saw it and added it.